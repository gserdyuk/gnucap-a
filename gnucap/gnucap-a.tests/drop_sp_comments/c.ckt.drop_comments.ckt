This is the title
*this is a comment
.options drop_spice_comments = yes
VIN 12 0 AC .5
EIN 0 9 12 0 1
M1 3 9 10 11 MOD1 W=59U L=10U
M2 4 12 10 11 MOD1 W=59U L=10U
M3 3 3 1 1 MOD2 W=8U L=10U
M4 4 3 1 1 MOD2 W=8U L=10U
M5 10 8 11 11 MOD1 W=51U L=10U
M6 5 4 1 1 MOD2 W=39U L=10U
M7 5 8 11 11 MOD1 W=949U L=10U
M8 8 8 11 11 MOD1 W=56U L=10U
IREF 1 8 10U
*M9 8 8 7 1 MOD2 W=10U L=10U
*M10 7 7 6 1 MOD2 W=11U L=10U
*M11 6 6 1 1 MOD2 W=10U L=10U
.MODEL MOD1 NMOS Level=2 VTO=1.0 KP=17U GAMMA=1.3 LAMBDA=0.01 PHI=0.7
.MODEL MOD2 PMOS Level=2 VTO=-1.0 KP=8U GAMMA=0.6 LAMBDA=0.02 PHI=0.6
CC 4 5 6PF
VCC 1 0 
* comment in the middle
+ DC +5
VSS 11 0 DC -5
.options drop_spice_comments = no
.options drop_spice_comments = no
*.AC DEC 10 1HZ 20M
*.plot ac vdb(5) vp(5)
*>.options out=200 vmin=-5.5 vmax=5.5 dampstrategy=2
*>.print op v(1 3 4 5 8 9 10 11 12)
.list
.op trace iterations
*>.stat notime
.END
