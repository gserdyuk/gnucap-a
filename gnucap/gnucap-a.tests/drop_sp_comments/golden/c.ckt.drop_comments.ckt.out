Gnucap gnucap-arails  2009.12.07 RCS 26.136 master 2010-09-25  bmm_semi_hsp, bm_model fix, back to bmm_semi
The Gnu Circuit Analysis Package
Never trust any version less than 1.0
Copyright 1982-2009, Albert Davis
Gnucap comes with ABSOLUTELY NO WARRANTY
This is free software, and you are welcome
to redistribute it under the terms of 
the GNU General Public License, version 3 or later.
See the file "COPYING" for details.
This is the title 
*this is a comment 
VIN ( 12 0 )  AC  0.5
EIN ( 0 9 12 0 )  1.
M1 ( 3 9 10 11 )  MOD1  l=10U w=59U
M2 ( 4 12 10 11 )  MOD1  l=10U w=59U
M3 ( 3 3 1 1 )  MOD2  l=10U w=8U
M4 ( 4 3 1 1 )  MOD2  l=10U w=8U
M5 ( 10 8 11 11 )  MOD1  l=10U w=51U
M6 ( 5 4 1 1 )  MOD2  l=10U w=39U
M7 ( 5 8 11 11 )  MOD1  l=10U w=949U
M8 ( 8 8 11 11 )  MOD1  l=10U w=56U
IREF ( 1 8 )  DC  10.u
.model MOD1 nmos2 ( level=2 kp=17U nfs=NA( 0.) delta=NA( 0.) vto=1.0 gamma=1.3 phi=0.7 lambda=0.01 tox= 100.n uo=NA( 600.) tpg=1 tnom=NA( 27.) fc=NA( 0.5) pb=NA( 0.8) cj=NA( 0.) mj=NA( 0.5)
+ cjsw=NA( 0.) mjsw=NA( 0.33) is=NA( 10.f) cgso=NA( 0.) cgdo=NA( 0.) cgbo=NA( 0.) ld=NA( 0.))
.model MOD2 pmos2 ( level=2 kp=8U nfs=NA( 0.) delta=NA( 0.) vto=-1.0 gamma=0.6 phi=0.6 lambda=0.02 tox= 100.n uo=NA( 600.) tpg=1 tnom=NA( 27.) fc=NA( 0.5) pb=NA( 0.8) cj=NA( 0.) mj=NA( 0.5)
+ cjsw=NA( 0.) mjsw=NA( 0.33) is=NA( 10.f) cgso=NA( 0.) cgdo=NA( 0.) cgbo=NA( 0.) ld=NA( 0.))
CC ( 4 5 )  6.p
VCC ( 1 0 )  DC  5.
VSS ( 11 0 )  DC -5.
*.AC DEC 10 1HZ 20M 
*.plot ac vdb(5) vp(5) 
#           v(1)       v(3)       v(4)       v(5)       v(8)       v(9)       v(10)      v(11)      v(12)     
 0.         0.         0.         0.         0.         0.         0.         0.         0.         0.        
-1.         5.        -15.398p   -15.398p   -15.398p    4.19Meg    0.        -4.031     -5.         0.        
-2.         5.         3.7462     1.235     -237.55    -3.0407     0.        -287.16    -5.         0.        
-3.         5.        -64.311    -35.995     26.218    -3.2728     0.        -4.5495    -5.         0.        
-4.         5.        -5.111     -4.9702     4.4986    -3.3371     0.        -4.0121    -5.         0.        
-5.         5.        -1.1825    -1.1793     4.095     -3.3999     0.        -0.2155    -5.         0.        
-6.         5.         1.202      1.2054     3.1832    -3.4033     0.        -95.871    -5.         0.        
-7.         5.         1.6327     1.6362     2.3714    -3.4033     0.        -5.5358    -5.         0.        
-8.         5.        -5.3688    -5.377      10.326    -3.4033     0.        -4.3507    -5.         0.        
-9.         5.        -2.9817    -3.152      4.6743    -3.4033     0.        -3.8452    -5.         0.        
-10.        5.         0.082097  -0.54247    4.0191    -3.4033     0.        -1.9291    -5.         0.        
-11.        5.         1.9127     1.2628     2.9325    -3.4033     0.        -98.853    -5.         0.        
-12.        5.        -4.0695    -3.0986     5.8552    -3.4033     0.        -4.2289    -5.         0.        
-13.        5.        -1.9079    -1.1251     4.3305    -3.4033     0.        -3.6772    -5.         0.        
-14.        5.         0.67053    1.446      3.4117    -3.4033     0.        -3.0105    -5.         0.        
-15.        5.         2.0063     2.5409    -4.1716    -3.4033     0.        -2.702     -5.         0.        
-16.        5.         2.5416     2.8485    -68.149    -3.4033     0.        -2.5851    -5.         0.        
-17.        5.         2.5878     2.815     -5.1307    -3.4033     0.        -2.5751    -5.         0.        
-18.        5.         2.6561     2.6793    -4.9816    -3.4033     0.        -2.5603    -5.         0.        
-19.        5.         2.658      2.6581    -4.9713    -3.4033     0.        -2.5599    -5.         0.        
-20.        5.         2.658      2.658     -4.9711    -3.4033     0.        -2.5599    -5.         0.        
-21.        5.         2.658      2.658     -4.9711    -3.4033     0.        -2.5599    -5.         0.        
-22.        5.         2.658      2.658     -4.9711    -3.4033     0.        -2.5599    -5.         0.        
 27.        5.         2.658      2.658     -4.9711    -3.4033     0.        -2.5599    -5.         0.        
Gnucap   System status
iterations: op=23, dc=0, tran=0, fourier=0, total=23
transient timesteps: accepted=0, rejected=0, total=0
nodes: user=9, subckt=0, model=0, total=9
dctran density=75.3%, ac density=75.3%
